src/Arachnid.ts

```diff
@@ -355,7 +361,8 @@ export default class Arachnid extends EventEmitter {
       url: response !== null ? response.url() : singlePageUrl,
       response: response !== null ? response : this.getErrorResponse(singlePageUrl, 'Unknown error'),
       extractedInfo,
-      depth: singlePageLink.depth
+      depth: singlePageLink.depth,
+      responseTime: pagePerformanceEntry.length > 0 ? Math.floor(pagePerformanceEntry[0].responseStart/10)/100: undefined
```

Comment:
1. Tend to see ms vs s as metric, what do you think?
2. I'm haven't find documentation better that [issue #417](https://github.com/puppeteer/puppeteer/issues/417) but the idea here do we want time difference/responseEnd?

Reply:
1. I kept the time as seconds (rounded for two decimal points), it is easier to read for the library users
2. I think it is better to use `responseStart` as it is (not the difference), as it indicates the **time to first byte**, more info about those can be found [here](https://developers.google.com/web/tools/chrome-devtools/network/understanding-resource-timing)

Reply:
Seems to me it differs a lot based on the response content and may vary from site to site, take alrai.com you will notice ~double value. 
Furthermore, I think the `responseStart` metric measures from the start time of incoming response reference to `startTime`